helix-cli/src/commands/integrations/helix.rs

```diff
@@ -153,7 +153,7 @@ impl<'a> HelixManager<'a> {
         Ok(())
     }
 
-    pub(crate) async fn deploy(&self, path: Option<String>, cluster_name: String) -> Result<()> {
+    pub(crate) async fn deploy(&self, path: Option<String>, cluster_name: String, dev_profile: Option<bool>) -> Result<()> {
```

Comment:
Again, this is special casing for a dev_profile, pass in the BuildMode here. Also, no need for it to be an Option<bool>, so don't do an Option<BuildMode> either, the cloud API takes in an Option<bool> for backwards compatibility only.